source/manage/hide-emojis.rst

```diff
@@ -0,0 +1,40 @@
+Hide emojis from the Emoji Picker
+=================================
+
+|all-plans| |self-hosted|
+
+.. |all-plans| image:: ../images/all-plans-badge.png
+  :scale: 30
+  :target: https://mattermost.com/pricing
+  :alt: Available in Mattermost Free and Starter subscription plans.
+
+.. |self-hosted| image:: ../images/self-hosted-badge.png
+  :scale: 30
+  :target: https://mattermost.com/deploy
+  :alt: Available for Mattermost Self-Hosted deployments.
+
+Emojis are small, digital images or icons used to communicate or express concepts such as emotions and physical gestures in messages. In addition to the default set of emojis that ship with Mattermost, users can supplement Mattermost standard emojis with by `uploading custom emojis <https://docs.mattermost.com/messaging/using-emoji.html#creating-custom-emojis>`__. 
+
+To hide a standard emoji in Mattermost, a System Admin must identify each emoji offending by name, and then rebuild their self-hosted instance of the Mattermost webapp to hide the emojis.
+
+.. note:: 
+
+    - The ability to remove default emojis from Cloud deployments is not supported.
+    - Removing a standard emoji from the Mattermost web app also removes it from all Mattermost clients, including the Desktop App and mobile apps. However, a hidden emoji doesn't stop users from using unicode characters to apply a hidden emoji visible to other Mattermost users.
```

Comment:
how does this happen? have anyone done this successfully?

Reply:
@coltoneshaw - I recall you taking @M-ZubairAhmed's draft content for a spin. Did you observe how mobile apps would behave following this rebuild operation?

Reply:
It's not removed from the mobile app, just the desktop and web app. I _think_ it's also removed from the reactions on the mobile app.

Reply:
mobile reactions are still present as well + the fact that all emojis are converted to unicode instead of using an image

Reply:
Confirmed what @enahum said above. When running this script it completely removes it from anything rendering our webapp, but does not remove it from the mobile app itself. 

You _can_ still insert this using the emoji keyboard on your desktop OS though. Example, on macOS i can copy 🤫 and paste it into Mattermost, which allows it to render.
![Screen Shot 2022-02-02 at 4 42 22 PM](https://user-images.githubusercontent.com/46071821/152242086-e933dcd5-1333-4e42-87ab-a5b6ba6ec2ef.png)

Reply:
View of the same interaction from the mobile client. 

![rn_image_picker_lib_temp_7b2ffbda-2557-4e72-a7ce-afca01874fde](https://user-images.githubusercontent.com/46071821/152242273-6d73f6e1-2213-4548-a870-a9206fe8b67e.png)
![rn_image_picker_lib_temp_0e57eeb0-d2b8-4c14-ba2a-93b39cbfae81](https://user-images.githubusercontent.com/46071821/152242277-49205103-e4b4-465b-bd7a-75176dfd4d7d.png)

Reply:
Hey @coltoneshaw @cwarnermm let's wait for this https://github.com/mattermost/mattermost-webapp/pull/9725 to get merged which makes it easy to add server side emoji file. Once that is merged I can update this pr to include those steps as well.

Reply:
Thanks, @M-ZubairAhmed!